cfme/tests/containers/test_reports.py

```diff
@@ -0,0 +1,201 @@
+# -*- coding: utf-8 -*-
+import re
+
+import pytest
+
+from cfme.containers.provider import ContainersProvider
+from cfme.intelligence.reports.reports import CannedSavedReport
+from cfme.fixtures.base import appliance
+from utils import testgen
+from utils.blockers import BZ
+
+
+pytestmark = [
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.meta(
+        server_roles='+ems_metrics_coordinator +ems_metrics_collector +ems_metrics_processor'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate([ContainersProvider], scope='function')
+
+
+@pytest.fixture(scope='module')
+def node_hardwares_db_data():
+
+    """Grabbing hardwares table data for nodes"""
+
+    db = appliance().db
```

Comment:
Again, why import and call this explicitly instead of just providing it as a parameter in your fixture definition?